stories/Molecules/Text/Authors/Authorcolumn/Authors.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './authors.scss';
+import { Heading } from '../../../../Atom/Base-typography/Heading/Heading';
+import { Authorimg } from '../../../../Atom/Images/Authorimage/Authorimages';
+
+export const Author = ({ data, width }) => {
+    return (
+        <div className="author-wrapper">
+            {data.map(function(item, index) {
+                return (
+                    <div className={['author__box', `author-${width}-col`].join(' ')} key={index}>
```

Comment:
@Shubham-Sharma-77 I think bem classname should be 'author-wrapper__box'

Reply:
@surbhi90 Ok